File: EvolutionUnity/Assets/Scripts/Facebook/FacebookUserInfoMy.cs
namespace com.littleteam.evon.serialization
{
    using Sfs2X.Protocol.Serialization;
    /// <summary>
    /// Инфо о пользователе фейсбука в моем представлении
    /// </summary>
    public class FacebookUserInfoMy : SerializableSFSType
    {
        public string Id = "";
        public string Email = "";
        public string FirstName = "";
        public string LastName = "";
        public int Gender = 0; //0 Unknow, 1 Male, 2 Female
        public string Locale = "";

        public string getDump()
        {
            return "Dump:\n" +
                Id + "\n" +
                Email + "\n" +
                FirstName + "\n" +
                LastName + "\n" +
                Gender.ToString() + "\n" +
                Locale;
        }
    }
}

File: ServerExtensions/EvolutionUtils/src/com/littleteam/evon/serialization/FacebookUserInfoMy.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package com.littleteam.evon.serialization;

import com.smartfoxserver.v2.protocol.serialization.SerializableSFSType;

/**
 * Информация о фейсбук пользователе
 * @author PanCrucian
 */
public class FacebookUserInfoMy implements SerializableSFSType {

    public String Id = "";
    public String Email = "";
    public String FirstName = "";
    public String LastName = "";
    public int Gender = 0; //0 Unknow, 1 Male, 2 Female
    public String Locale = "";

    /**
     * Все в строку
     * @return 
     */
    public String getDump() {
        return "Dump:\n"
                + Id + "\n"
                + Email + "\n"
                + FirstName + "\n"
                + LastName + "\n"
                + String.valueOf(Gender) + "\n"
                + Locale;
    }
}
